pkg/lint/lintersdb/manager.go

```diff
@@ -456,6 +458,12 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithSince("v1.36.0").
 			WithPresets(linter.PresetStyle).
 			WithURL("https://github.com/esimonov/ifshort"),
+
+		linter.NewConfig(golinters.NewIreturn(ireturnCfg)).
+			WithSince("v1.43.0").
+			WithPresets(linter.PresetStyle).
+			WithURL("https://github.com/butuzov/ireturn"),
+
```

Comment:
can you remove the extra empty lines and move the linter near the end of the list?